go/libraries/doltcore/branch_control/access.go

```diff
@@ -118,62 +96,8 @@ func (tbl *Access) Serialize(b *flatbuffers.Builder) flatbuffers.UOffsetT {
 
 	// Serialize the binlog
 	binlog := tbl.binlog.Serialize(b)
-	// Initialize field offset slices
-	databaseOffsets := make([]flatbuffers.UOffsetT, len(tbl.Databases))
```

Comment:
I don't see any corresponding changes to the flatbuffers table definitions. Probably should mark the unused fields deprecated at least? Any plan to get rid of them?

Reply:
They're still used, in full, by the `dolt_branch_namespace_control` table. It's not obvious by the PR, but this change only affects the `dolt_branch_control` table, which is where the majority of checks occur (all write operations check this table for example). The matching logic is straightforward and doesn't have any special rules, while the namespace table checks for the length of each match. The new tree structure does support length checks, but not against sections, only the entire tree. I need to find a decent way of porting the tree over, but since this table is _far_ less likely to be used (**requiring** manual insertion) and also far less likely to have a ton of branches, I'm punting on it for now.

Reply:
That said, the binlog is still useful for the `dolt_branch_control` table, and all pre-existing files already have it filled out, so it's an easy way to keep backward compatibility.